File: POTATOES.py
import math
test_cases = int(input())
primes = []


def check_prime(n):
    global primes
    if n in primes:
        return True
    for i in range(2, int(math.sqrt(n))+1):
        if n % i == 0:
            return False
    return True


for _ in range(test_cases):
    x, y = map(int, input().strip().split())
    sum = x+y
    addr = 1
    found = False
    while not found:
        local_sum = sum+addr
        res = check_prime(local_sum)
        if res is True:
            found = True
            print(addr)
        else:
            addr += 1

File: POTATOES.go
package main

import (
	"fmt"
	"math"
)

func checkPrime(n int) bool {
	for i := 2; i < (int(math.Sqrt(float64(n))) + 1); i++ {
		if n%i == 0 {
			return false
		}
	}
	return true

}

func main() {
	var testCases, x, y int
	fmt.Scanln(&testCases)
	for i := 0; i < testCases; i++ {
		fmt.Scanf("%d %d", &x, &y)
		sum := x + y
		addr := 1
		found := false
		for !found {
			localSum := sum + addr
			res := checkPrime(localSum)
			if res == true {
				found = true
				fmt.Println(addr)
			} else {
				addr++
			}
		}
	}
}
